mxnet/inference/buildspec.yml

```diff
@@ -58,7 +58,7 @@ images:
  BuildSMMXNetCPUInferencePy3DockerImage:
    <<: *INFERENCE_REPOSITORY
    build: &MXNET_CPU_INFERENCE_PY3 false
-   image_size_baseline: 2202
+   image_size_baseline: 5000
```

Comment:
The image baseline size doubled. Any reason why?

Reply:
Thanks for catching this. I am rebuilding images with a smaller baseline size.